Makefile

```diff
@@ -50,3 +50,11 @@ clear-cache:
 	docker rm -vf $$(docker ps -a -q) 2>/dev/null ; \
 	docker rmi -f $$(docker images -a -q) 2>/dev/null ; \
 	docker volume rm $$(docker volume ls -q) 2>/dev/null ; true
+
+stop-trial-runners:
```

Comment:
Why not a simpler command stop-trials ?
and just one command, if experiment is set then cancels that experiment one, otherwise cancels all.

Reply:
Will people be running multiple experiments at once ? Will that even work, if not, just a stop-experiment seems better ?

Reply:
is there an assumption that the runs are running in the background and don't block the terminal? For example, when I run `make run-XXX`, it blocks the terminal and I have to Ctl+C to stop it. I'm wondering it it'd make sense to have a dedicated utility - rather than make - for updating/controlling running experiments/tasks, instead of cluttering the Makefile. I guess now we don't have that many utility function so it
so fine :)

Reply:
> I'm wondering it it'd make sense to have a dedicated utility - rather than make - for updating/controlling running experiments/tasks, instead of cluttering the Makefile. 

Yes, I'm hoping that this would get much cleaner when we switch to the queue based architecture. In that case each task will be a single `docker run` (or `docker build`) command, each worker would work on one task at a time, and we wouldn't even need to run background jobs.

Reply:
> Why not a simpler command stop-trials ?

or even something like `stop-all-fuzzbench-containers`, if it stops not just runners but others too (like measurer)?